Source/commands/for_CommandCoordinator/when_handling_with_before_handle_callbacks.ts

```diff
@@ -2,46 +2,47 @@
  *  Copyright (c) Dolittle. All rights reserved.
  *  Licensed under the MIT License. See LICENSE in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
-import { CommandCoordinator } from '../CommandCoordinator';
+import sinon from 'sinon';
```

Comment:
Sinon, Chai and others should be included implicitly by the test pipeline and something we can just rely on

Reply:
Typescript complains that it does not exist at compile time, even though it does in runtime. It might require some tinkering to make it possible

Reply:
Is that what the global.d.ts is for?
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/global-d-ts.html

Reply:
I’ll look into it.